tools/lib/extractB3dm.js

```diff
@@ -86,3 +97,10 @@ function extractB3dm(b3dmBuffer) {
         glb : glbBuffer
     };
 }
+
+function alignGlb(buffer, byteOffset) {
+    if (byteOffset % 4 === 0) {
+        return buffer;
+    }
+    return Buffer.from(buffer);
```

Comment:
From what I can tell, this is just copying `buffer` in memory if `byteOffset` isn't aligned to 4 bytes. Is this so the b3dm buffer can be modified somewhere without damaging the glb? It might be worth documenting.